resources/compass/charts/kyma-environment-broker/templates/secrets.yaml

```diff
@@ -25,4 +25,19 @@ data:
   url: {{ .Values.serviceManager.url | b64enc | quote }}
   username: {{ .Values.serviceManager.username | b64enc | quote }}
   password: {{ .Values.serviceManager.password | b64enc | quote }}
+---
+apiVersion: v1
+data:
+  apiEndpoint: {{ .Values.avs.apiEndpoint }}
```

Comment:
in CI overrides you have a different values:

```
  kyma-environment-broker.avs.apiEndpoint.apiKey: "***"
  kyma-environment-broker.avs.apiEndpoint.clientId: "**"
  kyma-environment-broker.avs.apiEndpoint.oauthPassword: "****"
  kyma-environment-broker.avs.apiEndpoint.oauthTokenEndpoint: "*****"
  kyma-environment-broker.avs.apiEndpoint.oauthUserName: "****"
```

so this will not work, please adjust that before merging and please notify my when this will be merged as this PR is blocked by this change: https://github.com/kyma-project/kyma/pull/7689 :)

Reply:
I see, I'm fixing it